bin/starter-pack/_webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const defaultConfig = require( "@wordpress/scripts/config/webpack.config" );
+
+module.exports = {
+	...defaultConfig,
+	externals: {
+		'@woocommerce/components': [ 'window', 'wc', 'components' ],
+		'@woocommerce/csv-export': [ 'window', 'wc', 'csvExport' ],
+		'@woocommerce/currency': [ 'window', 'wc', 'currency' ],
+		'@woocommerce/date': [ 'window', 'wc', 'date' ],
+		'@woocommerce/navigation': [ 'window', 'wc', 'navigation' ],
+		'@woocommerce/number': [ 'window', 'wc', 'number' ],
+		'@woocommerce/settings': [ 'window', 'wc', 'wcSettings' ],
+	}
+};
```

Comment:
If you go with my suggestion earlier to incorporate the dependency extraction webpack plugin, you can enhance the config of that plugin here so it accounts for `@woocommerce/*` externals too.